packages/git/src/browser/git-base-widget.ts

```diff
@@ -101,11 +100,13 @@ export class GitBaseWidget<T extends { selected?: boolean }> extends VirtualWidg
 
     protected onAfterAttach(msg: Message): void {
         super.onAfterAttach(msg);
-        this.addKeyListener(this.node, Key.ARROW_LEFT, () => this.handleLeft());
-        this.addKeyListener(this.node, Key.ARROW_RIGHT, () => this.handleRight());
-        this.addKeyListener(this.node, Key.ARROW_UP, () => this.handleUp());
-        this.addKeyListener(this.node, Key.ARROW_DOWN, () => this.handleDown());
-        this.addKeyListener(this.node, Key.ENTER, () => this.handleEnter());
+
+        const changesOuterContainer = <HTMLElement>this.node.getElementsByClassName('changesOuterContainer')[0];
+        this.addKeyListener(changesOuterContainer, Key.ARROW_LEFT, () => this.handleLeft());
+        this.addKeyListener(changesOuterContainer, Key.ARROW_RIGHT, () => this.handleRight());
+        this.addKeyListener(changesOuterContainer, Key.ARROW_UP, () => this.handleUp());
+        this.addKeyListener(changesOuterContainer, Key.ARROW_DOWN, () => this.handleDown());
+        this.addKeyListener(changesOuterContainer, Key.ENTER, () => this.handleEnter());
```

Comment:
Hi, 
please bind to the element with the id kept in `scrollContainer` field.
Since the `changesOuterContainer` class is just used in the git-widget the keybindings don't work in git-diff-widget and git-history-widget anymore.

Reply:
Yea that makes sense, although I'm not sure how to fix it as for the git-history, if I use the scrollContainer id to attach something to it in the onAfterAttach call it doesn't seem to be attached under `this.node` yet... Feel free to modify my branch on the theia repo if you can fix it easily :)

Reply:
Because `renderCommitList()` is called after I believe...